### PR TITLE
Add TypeScript definition

### PR DESCRIPTION
**Pull request by 3846masa**
- Add TypeScript definition
- Run type check when `npm test`

Ref #56

### CODE REVIEW

File: types/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "commonjs",
+    "lib": ["esnext"],
+    // "allowJs": true,
```
Comment (SerayaEryn) [inline types/tsconfig.json]: I think the commented out compiler options can be removed.

**Review by SerayaEryn — CHANGES_REQUESTED** [review]
That's nice work. Having the tests for the types will be very useful.

I just left one comment; beside that it is ready for being merged.

File: types/tsconfig.json (hunk 1 above)
Comment (3846masa) [reply types/tsconfig.json]: Ok, I'll remove.

**Comment by 3846masa** [comment]
@SerayaEryn Thanks for your review.
I cleaned up tsconfig.json. Please re-review.

**Review by SerayaEryn — APPROVED** [review]

**Comment by SerayaEryn** [comment]
Thanks for your contribution.

**Pull request merged**